File: root_VS2012_TemplateBase/programs/C#/Samples/2CS_sample/CustCtrl_sample/Bean.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CustCtrl_sample
{
    public class Bean
    {
        public decimal AAA { set; get; }
        public DateTime BBB { set; get; }
        public string CCC { set; get; }

        public Bean(decimal aaa, DateTime bbb, string ccc)
        {
            this.AAA = aaa;
            this.BBB = bbb;
            this.CCC = ccc;
        }
    }
}

File: root_VS2013_TemplateBase/programs/VB/Samples/2CS_sample/CustCtrl_sample/Bean.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Public Class Bean
    Public Property AAA() As Decimal
        Get
            Return m_AAA
        End Get
        Set(ByVal value As Decimal)
            m_AAA = Value
        End Set
    End Property
    Private m_AAA As Decimal
    Public Property BBB() As DateTime
        Get
            Return m_BBB
        End Get
        Set(ByVal value As DateTime)
            m_BBB = Value
        End Set
    End Property
    Private m_BBB As DateTime
    Public Property CCC() As String
        Get
            Return m_CCC
        End Get
        Set(ByVal value As String)
            m_CCC = Value
        End Set
    End Property
    Private m_CCC As String

    Public Sub New(ByVal aaa As Decimal, ByVal bbb As DateTime, ByVal ccc As String)
        Me.AAA = aaa
        Me.BBB = bbb
        Me.CCC = ccc
    End Sub
End Class
